packages/augur-ui/src/modules/app/components/top-nav/top-nav.styles.less

```diff
@@ -170,6 +161,14 @@ button.AddFunds {
   }
 }
 
+.MigrateRep {
+  > button {
+    @media all and (min-width: @tablet-min) and (max-width: 800px) {
```

Comment:
why this ad-hoc media breakpoint? we shouldn't be deviating from the mocks and if the mocks have odd font-sizes between breakpoints the design should be adjusted instead. Also don't set `font-size` directly, use one of the helpers like `.text-9` or whatever (8 seems too small, I don't think we should be using anything that low as that's outside of the typography we have laid out, another thing to bring up to design). 

This is most likely an oversight with the design rather than a hard request.